diff --git a/dist/types/json-schema.d.ts b/dist/types/json-schema.d.ts
index 78ec842fa84017f3ced3c5b088c16cdd4b636f41..f62bd626611b83fa0369ecd66e2f3603a9d1ab7a 100644
--- a/dist/types/json-schema.d.ts
+++ b/dist/types/json-schema.d.ts
@@ -33,7 +33,9 @@ declare type UncheckedJSONSchemaType<T, IsPartial extends boolean> = (// these t
     type: JSONType<"boolean", IsPartial>;
 } : T extends readonly [any, ...any[]] ? {
     type: JSONType<"array", IsPartial>;
-    items: {
+    items:{
+        $ref: string;
+    }| {
         readonly [K in keyof T]-?: UncheckedJSONSchemaType<T[K], false> & Nullable<T[K]>;
     } & {
         length: T["length"];
@@ -45,7 +47,9 @@ declare type UncheckedJSONSchemaType<T, IsPartial extends boolean> = (// these t
     additionalItems: false;
 }) : T extends readonly any[] ? {
     type: JSONType<"array", IsPartial>;
-    items: UncheckedJSONSchemaType<T[0], false>;
+    items: UncheckedJSONSchemaType<T[0], false> | {
+        $ref: string;
+    };
     contains?: UncheckedPartialSchema<T[0]>;
     minItems?: number;
     maxItems?: number;
